api/server.go

```diff
@@ -11,20 +11,26 @@ import (
 	"github.com/YaleSpinup/cost-api/costexplorer"
 	"github.com/gorilla/handlers"
 	"github.com/gorilla/mux"
+	cache "github.com/patrickmn/go-cache"
 
 	log "github.com/sirupsen/logrus"
 )
 
+var (
+	//Org will carry throughout the api and get tagged on resources
```

Comment:
👀

Reply:
What do you want to adjust?  I can't tell - thx

Reply:
put a space before `Org`, after the `//`